README.adoc

```diff
@@ -0,0 +1,73 @@
+= Jenkins Infra on AWS
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:toc:
+:private_repo_name: terraform-states
+:private_repo_url: https://github.com/jenkins-infra/{private_repo_name}
+
+This repository is meant to hold documentation, tooling and other resources
+related to the link:https://www.jenkins.io/projects/infrastructure/[Jenkins Infrastructure Project] parts hosted in
+link:https://aws.amazon.com/[Amazon Web Services (AWS)].
+
+== Requirements
+
+In order to use this repository to provision the Jenkins infrastructure on AWS, you need:
+
+* An AWS account with the ability to assume the role `infra-admin` on the AWS account used for the Jenkins infrastructure.
```

Comment:
aws?

Reply:
This file shouldn't have been commited :/